### DISCUSSION TITLE
host.oast.fun not working like host@oast.fun but in burp collabarator it is working

### DISCUSSION THREAD

**Discussion by balook** [projectdiscovery/interactsh#957] [Q&A]
visit https://app.interactsh.com/#/

you will be having a host 
example  host.oast.fun 

try sending email to host@oast.fun but you dont receive any smtp 
but if you try samething in burpsuite it will be working fine

**Comment by Mzack9999** [comment]
The email domain should be the payload generated by interactsh-client, for example:

```console
$ interactsh-client

    _       __                       __       __
   (_)___  / /____  _________ ______/ /______/ /_
  / / __ \/ __/ _ \/ ___/ __ '/ ___/ __/ ___/ __ \
 / / / / / /_/  __/ /  / /_/ / /__/ /_(__  ) / / /
/_/_/ /_/\__/\___/_/   \__,_/\___/\__/____/_/ /_/

                projectdiscovery.io

[INF] Current interactsh version 1.2.0 (latest)
[INF] Listing 1 payload for OOB Testing
[INF] cr87dhmltlrqck7gd9t0www49qass1d1z.oast.me
```

The target email is `xxxx@cr87dhmltlrqck7gd9t0www49qass1d1z.oast.me` with `xxx` any value that you want

**Comment by balook** [comment]
If it generated xyz.oast.me I should use it as xyz@oast.me ... Not xxxx@xyz.oast.me

**Comment by balook** [comment]
With burp collab u can use like as I said

**Reply by Mzack9999** [reply]
The email length would be the same anyway, plus in this way you can use the username part to further customize the taint or exfiltrate info such as environment variables (depending on your target)
